packages/compiler/src/plugins.ts

```diff
@@ -80,23 +80,32 @@ export const processMyst = () => {
           (parent as Parent).children[index] = newNode;
           return index;
         } else if (mystType === "panels") {
-          const mdCards = parsePanel(value);
-          const htmlCards = mdCards.map((card: MystCard) => {
+          const panel = parsePanel(value);
+          const htmlCards = panel.cards.map((card: MystCard) => {
             let k: keyof MystCard;
+            let micromarkCard = {} as MystCard;
             for (k in card) {
-              card[k] = micromark(card[k]);
+              if (k !== "style") {
+                micromarkCard[k] = micromark(card[k]);
+              }
             }
-            return card;
+            if (card.style) {
+              micromarkCard = {...micromarkCard, style: card.style}
+            }
+            return micromarkCard;
           });
-          // create dummy object to access array properties with mustache
-          const cards = { cards: htmlCards };
           // parse each card
           const newNode = {
             type: "html",
             value: mustache.render(
-              `<Panels>
+              `<Panels panelStyle="{{{styles.container}}}">
                {{#cards}}
-               <Card>
+               <Card columnStyle="{{{style.column}}}"
+                     cardStyle="{{{style.card}}}"
+                     headerStyle="{{{style.header}}}"
+                     bodyStyle="{{{style.body}}}"
+                     footerStyle="{{{style.footer}}}"
```

Comment:
Ah, I see the problem you were talking about now. I bet you anything you could serialize `style` as a JSON string, then pass it directly as a style property. Example:

https://svelte.dev/repl/7e9af063c80a4dcca87ce9cfd4318283?version=3.44.3

Reply:
Managed with https://github.com/irydium/irydium/pull/213/commits/f322bda0f8dc40629a44900e19980afc30655252